django/santropolFeast/santropolFeast/templates/system/_button_export.html

```diff
@@ -2,10 +2,6 @@
     <div class="ui button">Export</div>
     <div class="ui floating dropdown icon button" tabindex="0">
         <i class="dropdown icon"></i>
-        <div class="menu transition hidden" tabindex="-1">
-            <div class="item">
-                <i class="dropdown icon"></i>
-                <span class="text">Export</span>
                 <div class="menu transition hidden">
                     <div class="header">Format</div>
                     <div class="item"><a href="?format=csv">CSV</a></div>
```

Comment:
You need this: 
`<div class="item"><a href="{{request.get_full_path}}&format=csv">CSV</a></div>`